schemas/content/directory.schema.json

```diff
@@ -16,6 +16,10 @@
           "format": "uri",
           "const": "https://ns.adobe.com/xdm/content/directory"
         },
+        "_page": {
+        	  "$ref": "https://ns.adobe.com/xdm/content/directory-page#/definitions/page",
+          "description": "Page info, included if this resource is part of paginated result."
+        },
```

Comment:
_page included in directory definition. Not included in parent, content definition, as pagination does not apply for all 'content' related resources.